jenkins/scripts/integration_test_setup.sh

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env bash
+
+set -eu
+
+# Description:
+#   Runs the integration tests for metal3-dev-env in an executer vm
+#   Requires:
+#     - source stackrc file
+#     - openstack ci infra should already be deployed.
+#     - environment variables set:
+#       - AIRSHIP_CI_USER: Ci user for jumphost.
+#       - AIRSHIP_CI_USER_KEY: Path of the CI user private key for jumphost.
+# Usage:
+#  integration_test.sh
+#
+
+CI_DIR="$(dirname "$(readlink -f "${0}")")"
+
+# shellcheck disable=SC1090
+source "${CI_DIR}/utils.sh"
+
+DISTRIBUTION="${DISTRIBUTION:-ubuntu}"
+if [ "${DISTRIBUTION}" == "ubuntu" ]
+then
+  IMAGE_NAME="${CI_METAL3_IMAGE}"
+  BASE_VOLUME_NAME="metal3-ubuntu"
+else
+  IMAGE_NAME="${CI_METAL3_CENTOS_IMAGE}"
```

Comment:
@maelk actually I do not see anything else than this condition where Centos is included in this PR, I can remove the else from it, and we can proceed with merging this PR, what do you think?

Reply:
I think only removing the condition for Centos here is not enough. Because,  that will only ensure not to export `IMAGE_NAME` & `BASE_VOLUME_NAME ` vars. However,  you have rest of the logic related to volume that runs no matter what the distribution is. For example `# Wait for a test executer volume to be available...`. So, you will need to adjust the logic in a way that volume related commands that you are adding here run only for Ubuntu.